packages/e2e-tests/puppeteer/helpers.ts

```diff
@@ -117,27 +124,36 @@ class FakeMetaMask implements dappeteer.Dappeteer {
 }
 
 export async function setupFakeWeb3(page: Page, ganacheAccountIndex: number): Promise<void> {
-  const ganachePort = process.env.GANACHE_PORT || 8545;
-
   // TODO: This is kinda ugly but it works
   // We need to instantiate a web3 for the wallet so we import the web 3 script
   // and then assign it on the window
   const web3JsFile = fs.readFileSync(path.resolve(__dirname, 'web3/web3.min.js'), 'utf8');
   await page.evaluateOnNewDocument(web3JsFile);
   await page.evaluateOnNewDocument(`
-    window.web3 = new Web3("http://localhost:${ganachePort}");
+    window.web3 = new Web3(new Web3.providers.HttpProvider("${RPC_ENDPOINT}"));
+    ${
+      TARGET_NETWORK === 'ropsten'
+        ? 'window.web3.eth.accounts.wallet.add("0xccb052837ccafb700e34c0e0cc0f3e5fbee8f078f3fe6b4e5950c7c8acaa7bce");'
```

Comment:
Should this be configured via an ENV variable, with default value `'0xccb052837ccafb700e34c0e0cc0f3e5fbee8f078f3fe6b4e5950c7c8acaa7bce'`?